### ISSUE TITLE
Categories Menu: Unable to access using a keyboard

### ISSUE THREAD

**Issue by shaunandrews** [WordPress/wporg-news-2021#244]
<img width="155" alt="image" src="https://user-images.githubusercontent.com/191598/151033507-1de4e6dd-e381-4c0d-9527-0ea50f505744.png">

I can't seem to set `:focus` on the Categories button using my keyboard.

**Comment by ryelle** [comment]
What browser are you using? I'm able to navigate with my keyboard and toggle the element fine in Firefox & Safari.

Screen recording from FF:

https://user-images.githubusercontent.com/541093/151037530-4ba1b1f2-e700-4da1-9c13-54d201ef1ad6.mov

**Comment by shaunandrews** [comment]
Ah, its really hard to see one some of the interior category pages, like on `/category/community/`

<img width="231" alt="image" src="https://user-images.githubusercontent.com/191598/151042014-8e1db9f0-6cf5-47e1-9283-af8325ad6a07.png">

**Comment by ryelle** [comment]
Hm, this is what I see on Community:

<img width="294" alt="Screen Shot 2022-01-25 at 2 07 19 PM" src="https://user-images.githubusercontent.com/541093/151042640-1063f7ed-e292-4880-9ef4-bbc39e580e90.png">

**Comment by shaunandrews** [comment]
Huh, maybe something with Safari and my machine; I can see it now, but previously I couldn't. I can't seem to reproduce now...
